docker/aws.env

```diff
@@ -1,13 +1,13 @@
 RUST_BACKTRACE="1"
 
 # The region of your object storage bucket
-S3_COMPATIBLE_REGION=your_region
+AWS_REGION=your_region
 
 # The endpoint URL for your object storage
-S3_COMPATIBLE_ENDPOINT=your_endpoint
+OBJECT_STORAGE_ENDPOINT=your_endpoint
```

Comment:
minor: This env is intended to be used only for S3 enpoint. I think `RW_S3_ENDPOINT` is a better naming.